pontoon/sync/core.py

```diff
@@ -20,6 +21,27 @@
 
 log = logging.getLogger(__name__)
 
+def run_only_one(func):
```

Comment:
The naming for this is actually really hard. I think it should have the word "task" in it to signify it's only for celery tasks.

After asking around I think `serial_task` is a decent name for this.